index.md

```diff
@@ -19,7 +19,11 @@ This conference is open to everyone!
 
 ## What does it cost to attend? Where do I register?
 
-Registration information is still being figured out - we'll post updates here!
+Registration opens Mon Nov 13 @ 6pm EST. Tickets are free and first come, first served. We will post the link here when registration opens! Information about all the speakers and talks will be posted by Nov 11.
```

Comment:
Answer the questions in the same order as they are posed. Also the opening day of registration isn't in the question above.